packages/components/src/components/navigation-item/navigation-item.ts

```diff
@@ -10,8 +10,8 @@ import SolidElement from '../../internal/solid-element';
 
 /**
  * @summary Flexible button / link component that can be used to quickly build navigations. Takes one of 3 forms: link (overrides all other if 'href' is provided), button (default), or accordion (if 'children' slot defined).
- * @status experimental
- * @since 1.12.0
+ * @status stable
+ * @since 1.15.0
```

Comment:
Is this intended?

Reply:
yes, the version was incorrect --- regarding status, it is not clear to me how you determine that, imo this is now stable and not experimental

Reply:
everything is stable unless we specifically announce something as experimental